src/client/src/containers/AzureLoginModal/strings.ts

```diff
@@ -0,0 +1,40 @@
+import { defineMessages } from "react-intl";
+
+export const strings = defineMessages({
+    freeAccountAnswer: {
+        defaultMessage: "All you need is a phone number, a credit or debit card, and a GitHub account or Microsoft account username (formerly Windows Live ID).",
+        id: "auzreLoginModal.freeAccountAnswer"
```

Comment:
Nit: isn't it azureLoginModal, happens throughout the file.

Reply:
Good Catch! Thanks :D